src/test/resources/net/sf/jabref/importer/fileformat/BiblioscapeImporterTestStandard.bib

```diff
@@ -1,14 +1,13 @@
-@article{,
-abstract = {The mild base-promoted reaction of methyl 2-phenyl-1-azirine-3-acetate (1) with aldehydes and acetone provides a new and simple route to the 3-oxazolines 5, which are formed in good yields by the electrophilic trapping of an imino anion produced by C-N bond cleavage in the 1-azirine enolate intermediate 6. Chloranil oxidation of 5 containing an aromatic substituent at C-2 affords oxazoles 7, while reaction of 5 containing an aliphaticgroup at C-2 produces 5-methylene-3-oxazolines 8 and 5-spiro-2-oxazolines 9 in addition to 7. [References: 11]},
-address = {Reprint available from: Kascheres A UNIV ESTADUAL CAMPINAS INST QUIM CP 6154 BR-13083970 SAO PAULO BRAZIL UNIV ESTADUAL CAMPINAS INST QUIM BR-13083970 SAO PAULO BRAZIL},
-author = {Sa, M. CM.; Kascheres, A.},
-comment = {Quaternary Title: Chemistry.},
-isbn = {0022-3263},
-journal = {Journal of Organic Chemistry},
-language = {English},
-number = {11},
-pages = {3749--3752},
-title = {Electronically Mediated Selectivity IN Ring Opening OF 1-azirines - THE 3-x Mode - Convenient Route To 3-oxazolines},
-volume = {61},
-year = {1996}
+@misc{,
```

Comment:
Maybe the filename suffix "Standard" is bad. When reading the test, I might think, that the importer always imports to `@misc` type. Why not calling it `UnkownType`, because the test is for an unknown type now.